Scripts/Morphology_main.py

```diff
@@ -63,70 +54,95 @@ def default(self, obj):
             return obj.tolist()
         return json.JSONEncoder.default(self, obj)
 
-def main(input_file, metadata_file, output_measure, output_landmark, output_presence, 
-         output_lm_image=None):
+def argument_parser():
+    parser = argparse.ArgumentParser(description='Extract information from segmented fish image such as presence absence,\
+                                     landmarks, measures.')
+    parser.add_argument('input_image', help='Path of segmented fish image. Format PNG image file.')
+    parser.add_argument('output_presence', help='Path of output presence absence table. Format JSON file.')
+    
+    
+    parser.add_argument('--metadata', help='Path of input drexel_metadata_formatter. Format JSON metadata file.')
```

Comment:
It seems this refers to a metadata _file_, not a metadata _formatter_. Perhaps you mean to say that this is the path to the metadata file in  JSON format, structured as output by drexel_metadata_formatter?

Reply:
change as https://github.com/hdr-bgnn/Morphology-analysis/commit/6b2bdc1136c25945cd86cedfd91996ea1c319992